convert/src/vertex_to_bd/badgerdoc_format/badgerdoc_format.py

```diff
@@ -0,0 +1,62 @@
+from pathlib import Path
+from typing import Optional
+
+from ..models.bd_annotation_model import BadgerdocAnnotation
+from ..models.bd_tokens_model import Page
+from ..models.vertex_models import LabelStudioModel
+from .annotation_converter import AnnotationConverter
+from .pdf_renderer import PDFRenderer
+from .plain_text_converter import PlainTextToBadgerdocTokenConverter
+
+
+class BadgerdocFormat:
+    def __init__(
+        self,
+        page_width=595,
+        page_border_offset=15,
+        font_height=11,
+        font_width=7,
+        line_spacing=2,
```

Comment:
Would be nice to have a comment or readme somewhere explaining these default values. I think we do not need them here.

Reply:
fixed